djreact/djangoapps/trips/models.py

```diff
@@ -0,0 +1,154 @@
+import json
+from datetime import datetime
+
+from django.db import models
+from django.db.models import permalink
+
+
+class Host(models.Model):
+    """
+    Trip host model.
+
+    This model contains the information for the trip hosts who are organizing
+    trips.
+    """
+    name = models.CharField(max_length=30)
+    slug = models.SlugField(max_length=50, null=True, blank=True)
```

Comment:
If we intend to use slug in URL's it should not be an optional field. Also why are we adding this slug field?

Reply:
Just to be on safe end. I think i should update the code to make it mandatory.